plugins/tpm/fu-self-test.c

```diff
@@ -84,15 +88,15 @@ fu_tpm_device_2_0_func(void)
 	if (tpm_server_running == NULL && (getuid() != 0 || geteuid() != 0)) {
 		g_test_skip("TPM2.0 tests require simulated TPM2.0 running or need root access "
 			    "with physical TPM");
-		return;
+		goto done;
 	}
 #endif
 
 	if (!fu_device_setup(FU_DEVICE(device), &error)) {
 		if (tpm_server_running == NULL &&
 		    g_error_matches(error, FWUPD_ERROR, FWUPD_ERROR_NOT_FOUND)) {
 			g_test_skip("no physical or simulated TPM 2.0 device available");
-			return;
+			goto done;
```

Comment:
we don't use goto in the fwupd project, so just manually add g_unsetenv("FWUPD_FORCE_TPM2"); here.